moon/config/moon-round3.json

```diff
@@ -3,87 +3,90 @@
   "robot": {
     "modules": {
       "app": {
-          "driver": "moon.controller_round3:SpaceRoboticsChallengeRound3",
```

Comment:
in general I hate reformating as in the diff is completely hides what changed and can lead to conflicts with merge of other PR. As discussed we should define what format we expect in JSON and this will no longer happen

Reply:
Of course but it was a mess formatting-wise so one time reformatting was needed one way or another. Hopefully the 2-space convention we agreed on for JSON is the one we are happy with going forward.